snowpack/src/commands/dev.ts

```diff
@@ -1296,7 +1296,8 @@ export async function startDevServer(commandOptions: CommandOptions): Promise<Sn
   const ips = Object.values(os.networkInterfaces())
     .reduce((every: os.NetworkInterfaceInfo[], i) => [...every, ...(i || [])], [])
     .filter((i) => i.family === 'IPv4' && i.internal === false)
-    .map((i) => i.address);
+    .map((i) => i.address)
+    .slice(0, 1);
```

Comment:
Should I update `getServerInfoMessage` to take `remoteIp` instead of `ips` 🤔?

Reply:
Yup, why don't you go ahead and make that change as well if you feel comfortable!

Reply:
Done